tools/tasks/seed/karma.start.ts

```diff
@@ -1,12 +1,14 @@
 import * as karma from 'karma';
 import { join } from 'path';
 
+import { APP_SRC } from '../../config';
+
 /**
  * Executes the build process, running all unit tests using `karma`.
  */
 export = (done: any) => {
   new (<any>karma).Server({
-    configFile: join(process.cwd(), 'karma.conf.js'),
+    configFile: join(process.cwd(), APP_SRC, 'karma.conf.js'),
```

Comment:
Currently `karma.conf.js`gets copied over to `dist/dev`.
Question: Should the `configFile` property point towards the `karma.conf.js` in `dist/dev` or the source `karma.conf.js` in `src/browser`?